forest/sycamore/tests/test_functions.py

```diff
@@ -98,9 +93,26 @@ def test_agg_changed_answers_summary():
     ca_detail, ca_summary = agg_changed_answers_summary(
         survey_settings_path, agg_data
     )
-    assert ca_detail.shape[0] == 5
+    assert ca_detail.shape[0] == 8
     assert ca_detail.shape[1] == 14
-    assert ca_summary.shape[0] == 4
+    assert ca_summary.shape[0] == 7
+    assert ca_summary.shape[1] == 9
+
+
+def test_agg_changed_answers_summary_no_config():
+    # make sure that changed_answers_summary is the same if we aggregate the
+    # data using the no config function
+    study_dir = os.path.join(TEST_DATA_DIR, "sample_dir")
+    survey_settings_path = os.path.join(
+        TEST_DATA_DIR, "sample_study_surveys_and_settings.json"
+    )
```

Comment:
It might be worth converting `study_dir` and `study_settings_path` (and similar variables) to fixtures.
https://docs.pytest.org/en/6.2.x/fixture.html#fixtures

Reply:
I believe that these paths are maybe handled just as well as global strings? Or is there any reason why they should be fixtures?

I did use fixtures for the aggregated data, which gets generated in multiple functions.

Reply:
In this case global variables are fine too. Fixtures are useful when there is additional logic or file objects. See Fixtures section for more details: https://www.linuxjournal.com/content/python-testing-pytest-fixtures-and-coverage